.bazelrc

```diff
@@ -16,10 +16,6 @@ build --spawn_strategy=local
 build --verbose_failures
 build --workspace_status_command=envoy/bazel/get_workspace_status
 build --xcode_version=13.2.1
-build --host_javabase=@bazel_tools//tools/jdk:remote_jdk11
```

Comment:
There is a new way to provide these values on Bazel 5.0 if you want to preserve this. At minimum you might want to provide the `java_language_version` and `tool_java_language_version` just to make sure that it's consistent moving forward.

Ex:
```
build \
  --java_language_version=8 \
  --tool_java_language_version=8 \
  --java_runtime_version=lyft_jdk11 \
  --tool_java_runtime_version=lyft_jdk11
```

Reply:
These are actually defined in Envoy now, and we import that `.bazelrc` file: https://github.com/envoyproxy/envoy/blob/708742e9e8ec44e634a02d16eee0dce826130298/.bazelrc#L19-L20